packages/cli/src/oclif/commands/register.js

```diff
@@ -32,7 +138,42 @@ RegisterCommand.args = [
       "Your integrations's public title. Asked interactively if not present.",
   },
 ];
-RegisterCommand.flags = buildFlags();
+
+RegisterCommand.flags = buildFlags({
+  commandFlags: {
+    desc: flags.string({
+      char: 'D',
+      description:
+        'A sentence describing your app in 140 characters or less, e.g. "Trello is a team collaboration tool to organize tasks and keep projects on track."',
```

Comment:
@eliangcs Jorge and I [discussed enforcing this limit in the CLI last week](https://zapier.slack.com/archives/C5Z9BP4U9/p1675095825596589?thread_ts=1675095407.301199&cid=C5Z9BP4U9). I can add custom validation to check the character limit of the provided description. I agree that we may change the character limit in the future, but there is a desire to enforce the current limit now. Are you comfortable with me proceeding with a 140 character limit validator?

Reply:
> Are you comfortable with me proceeding with a 140 character limit validator?

I'm good with that as long as the 140-character limit is done on the CLI client. If we want to validate that on the server side, I'd say we let the server give the error and the CLI client shouldn't mention anything about the limit.

Reply:
@eliangcs I started using a constant for the max allowed description length (in 3b577478095c8916a83eb643b3e96af126c60133) - hopefully this will help make it easier to update if we ever need to change this value.